src/index.css

```diff
@@ -214,34 +216,53 @@
   .scrollTop {
     left: 43%;
   }
+  .app__header {
+    position: sticky;
+    top: 0;
+  }
 }
 
 /* Small devices (portrait tablets and large phones, 600px and up) */
 @media only screen and (min-width: 600px) {
   .scrollTop {
     left: 45.6%;
   }
+  .app__header {
+    position: sticky;
+    top: 0;
+}
 }
-
 /* Medium devices (landscape tablets, 768px and up) */
 @media only screen and (min-width: 768px) {
   .scrollTop {
     left: 46.6%;
   }
+  .app__header {
+    position: sticky;
+    top: 0;
+  }
 }
 
 /* Large devices (laptops/desktops, 992px and up) */
 @media only screen and (min-width: 992px) {
   .scrollTop {
     left: 47.2%;
   }
+  .app__header {
+    position: sticky;
+    top: 0;
+  }
 }
 
 /* Extra large devices (large laptops and desktops, 1200px and up) */
 @media only screen and (min-width: 1200px) {
   .scrollTop {
     left: 47.7%;
   }
+  .app__header {
```

Comment:
I don't think you need to copy paste this property at every query. Find a better solution(remember the DRY or Don't Repeat Yourself principle)